apps/web/app/api/assets/route.ts

```diff
@@ -26,7 +27,7 @@ export async function POST(request: Request) {
     });
   }
   const formData = await request.formData();
-  const data = formData.get("image");
+  const data = formData.get("file");
```

Comment:
For backward compatibility reasons, especially with phones which don't get release immediately, we'll need to keep accepting the `image` field for a while.
```suggestion
  // TODO: Deprecate the `image` field in a couple of releases
  const data = formData.get("file") ?? formData.get("image");
```